audit_exceptions/provided_by_macos_depends_on_allowlist.json

```diff
@@ -1,6 +1,7 @@
 [
   "apr",
   "apr-util",
+  "libarchive",
```

Comment:
I don't think that's enough, Homebrew should explicitly disallow `uses_from_macos libarchive`?

Reply:
At this stage there is no reason, and it would be a brew change.

Reply:
@fxcoudert Should this `brew` change not happen right now or ever? Can you elaborate a bit more on reasoning here? Can be post-merge.

Reply:
There's an open issue for this at https://github.com/Homebrew/brew/issues/18264, let's continue discussion there.

Marking this as resolved to allow this PR to merge.